app/views/pages/use-cases/implementing-social-media-preview-cards.liquid

```diff
@@ -115,6 +115,78 @@ After:
 
 <img src="{{ 'images/use-cases/social-media-cards/after.jpg' | asset_url }}" alt="Twitter card after adding OpenGraph meta tags" />
 
-## Other resources
+### Other resources
+
+If you wish to learn more about the topic, read: [Everything you ever wanted to know about unfurling but were afraid to ask /or/ How to make your site previews look amazing in Slack](https://medium.com/slack-developer-blog/everything-you-ever-wanted-to-know-about-unfurling-but-were-afraid-to-ask-or-how-to-make-your-e64b4bb9254)
+
+## Structured data
+
+After you have prepared your OpenGraph metadata, it is also a good idea to add so called **structured data** into your website.
+
+Structured data helps search engines find and understand your content and website. It is organized and tagged with specific groups of text that help search engines understand the context of that information and can return accurate results to searchers. 
+
+This metadata is sometimes used by websites to generate rich snippets instead of OpenGraph or an addition. Additionally, this data is used by search engines to populate their rich snippets. Using structured data you can describe your content in detail. Because of that, it is more flexible. It is possible to describe the relation between entities (author, image, website). The more correct details you provide (within reason), the better.
```

Comment:
It's not clear to me what "an addition" means in this sentence, so I couldn't edit it. Please rephrase to make it clearer. Thank you!

Reply:
Some systems take advantage of all the data in metadata, they gather everything (opengraph, content, strudcutred data) in one place and then form their output based on what they've got. Because not website has all of them (or one specific one), they try to do a good job anyway.

Reply:
^ We can just remove it as "too much detail" :)

Reply:
Thank you!